tests/gui/pages/main.spec.js

```diff
@@ -26,9 +27,17 @@ const availableDrives = require('../../../lib/shared/models/available-drives')
 const selectionState = require('../../../lib/shared/models/selection-state')
 require('angular-mocks')
 
+// Mock HTML requires by reading from the file-system
+// eslint-disable-next-line node/no-deprecated-api
+require.extensions['.html'] = (module, filename) => {
```

Comment:
Is this needed for this particular PR? (I think this is more of a Webpack change)

Reply:
I swear I removed that from the patch file... good catch though!